### ISSUE TITLE
Lua Error can't open Combuctor

### ISSUE THREAD

**Issue by Coffeechan** [Jaliborc/Bagnonium#189]
**Running Software (issues missing this information will be deleted):**
 - Addon version: 9.2.1-3.4.0
 - Server patch: WotLK 3.4.1
 -

 **Have you read the changelog? (please don't waste our time)**
 Mark if read. Yesh

**Describe the bug**

**To Reproduce**
Steps to reproduce the behaviour:
1. Login
2. click bag

**Expected behaviour**
When I click to open my bags nothing pops up

**Screenshots**

**Error Logs**
28x Usage: self:SetCheckedTexture(asset)
[string "=[C]"]: in function `SetCheckedTexture'
[string "@Combuctor/addons/main/qualityButton.lua"]:42: in function `New'
[string "@Combuctor/addons/main/qualityFilter.lua"]:32: in function `AddQualityButton'
[string "@Combuctor/addons/main/qualityFilter.lua"]:16: in function `New'
[string "@Combuctor/addons/main/frame.lua"]:37: in function `New'
[string "@Combuctor/common/api/frames.lua"]:44: in function `New'
[string "@Combuctor/common/api/frames.lua"]:81: in function `Show'
[string "@Combuctor/common/features/autoDisplay.lua"]:176: in function `OpenAllBags'
[string "@ArkInventory/Core/ArkInventory.lua"]:11200: in function <ArkInventory/Core/ArkInventory.lua:11020>
[string "=(tail call)"]: ?

**Additional context**
Cant pull up bag

**Comment by TitussPulo** [comment]
Same problem, Combuctor does not work with Phase 2 patch of WOTLK - I would love for it to work again as it is the best.

**Comment by Bakaflosama** [comment]
Yes, same as well as all guildmates on WoTLK that use Combuctor. Bagnon is fine now but i rlly prefer combuctor.

**Comment by TitussPulo** [comment]
Yes please someone fix combuctor.  It’s way better than bagnon

**Comment by SirDilferdIII** [comment]
I miss this addon, it's been broken since phase 2 launched. Bagnon not the same :C

**Comment by MyTechnoHunter** [comment]
release for WoTLK please test, report any issues either here or the discord

**Comment by TitussPulo** [comment]
This is great news, I will surely test and report! Thank you so much!

**Comment by SirDilferdIII** [comment]
It work well so far, thanks for fix C:

**Comment by TitussPulo** [comment]
Fantastic! It works! Thank you so much

**Comment by xiibo** [comment]
Any chance to get this working in retail?
